private/pkg/protosource/location_store.go

```diff
@@ -23,10 +23,8 @@ import (
 type locationStore struct {
 	sourceCodeInfoLocations []*descriptorpb.SourceCodeInfo_Location
 
-	pathToLocation               map[string]Location
-	pathToSourceCodeInfoLocation map[string]*descriptorpb.SourceCodeInfo_Location
-	locationLock                 sync.RWMutex
-	sourceCodeInfoLocationLock   sync.RWMutex
```

Comment:
Was there any need to modify it?

Reply:
It turns out there wasn't. I thought there would be at first, to accommodate the logic for "fuzzy search" for a match path for a custom option. But once I was able to make heads and tails of this code, I realized I didn't need to add it and that the new method would be just as well off falling back to a slow (no cache) search if it can't find an exact match.

So this really just code hygiene improvement -- just following the "boy scout rule". It's much easier to read and understand at a glance now IMO.

Reply:
I'll be the cranky old person and say I never understand onces, and the locks are easier for me to reason about because they use actual tried-and-true concurrency primitives, but alas.